ToolBox/Visualizer/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "composer-dll-directory": "../.",
+  "data-folder": "../../../../Data",
+  "default-output-folder": "../output_folder"
```

Comment:
Moar config files!

Reply:
We can remove this and pass the values as arguments to the script.